cmd/dep/status.go

```diff
@@ -49,6 +55,7 @@ func (cmd *statusCommand) Register(fs *flag.FlagSet) {
 	fs.BoolVar(&cmd.detailed, "detailed", false, "report more detailed status")
 	fs.BoolVar(&cmd.json, "json", false, "output in JSON format")
 	fs.StringVar(&cmd.template, "f", "", "output in text/template format")
+	fs.StringVar(&cmd.output, "o", "output.svg", "output file")
```

Comment:
Instead of `-o`, let's have `-dot`, then simply compute dot output as a string and print that to stdout. That'll be much more flexible for end-users.